PiAUISuite/VoiceCommand/commands.conf

```diff
@@ -1,7 +1,7 @@
 #This is the default config file
 #These are the special options you can set (remove the #)
 #!verify==1
-!keyword==mascot
+!keyword== mascot
```

Comment:
I don't think putting spaces between the key value pair is legal syntax for a `conf` file. Should be
```
key==value
```

Reply:
Got it